### PR TITLE
JEDI-1474 - extricate from safarijv

### PR DESCRIPTION
**Pull request by SQLCODE917**
https://intranet.oreilly.com/jira/browse/JEDI-1474

Migrate from safarijv to @oreillymedia in github. Matt Hare had requested all to vacate safarijv.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,282 @@
 # MetaReduxForm
 
-The MetaReduxForm module [tell us what this module does.]
+You define the inputs, they assemble into a form.
 
+## Features
```
Comment (kelvinhammond) [inline README.md]: Thank you very much for the documentation.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,282 @@
...
+### Overwriting Default From Components
+
+The HTML `<SELECT>`, for example, is a bothersome component to style - both due to it's default browser styles and due to the steps you need to take in order to target the subcomponents with CSS rules.
+In order to merge it successfully into our form component and into the underlying `redux-form` flow, we will be using a Closure which containes an Adapter.
+The outer function closes over the part of the `manifest` that describes the component we are replacing.
+The inner function adheres to the [redux-form Field API](https://redux-form.com/7.4.2/docs/api/field.md/) and adapts it to the API of your overwriting component.
+
+```javascript
+import React from 'react';
+import { Select } from '@oreilly/shape-react-core';
```
Comment (tairemad) [inline README.md]: we want to use select from design system

File: README.md [hunk 3]
```diff
@@ -1,7 +1,282 @@
...
+  };
+
+  return connect()(RemoteSubmitButton);
+};
+
+```
+
+```javascript
+import React from 'react';
+import { Button } from '@oreilly/shape-react-core';
```
Comment (tairemad) [inline README.md]: Again we want to use from design system

Comment (SQLCODE917) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (SQLCODE917) [reply README.md]: Done

File: .babelrc.js [hunk 4]
```diff
@@ -26,7 +26,23 @@ module.exports = function (api) {
 
   // Start with whatever is defined in the chassis-core config.
   const config = {
-    ...getToolConfig('babel')
+    ...getToolConfig('babel'),
+    presets: [
+      "@babel/preset-react",
```
[the remark this answers is not in the document]
Comment (SQLCODE917) [reply .babelrc.js]: Done

**Pull request merged**